.ci/docker/requirements-ci.txt

```
@@ -263,7 +263,8 @@ unittest-xml-reporting<=3.2.0,>=2.0.0
 #Pinned versions:
 #test that import:
 
-lintrunner==0.10.7
+#TODO: wheel not found on aarch64, and source build requires rust
+#lintrunner==0.10.7
```

Do not disable it, just constrain to x86_64 (according to https://peps.python.org/pep-0496/#strings )
```suggestion
#TODO: wheel not found on aarch64, and source build requires rust
lintrunner==0.10.7 ; platform_machine == "x86_64"
```

---

thank you, i wasn't aware of this syntax.